frontend/src/components/Navbar.js

```diff
@@ -46,6 +53,32 @@ function CourseTableNavbar({ isLoggedIn }) {
     window.location.pathname = '/';
   };
 
+  // DarkMode or LightMode
+  setFetchMethod(window.fetch);
+
+  // Checks user preferences for dark mode
+  if (!window.localStorage.getItem('darkmode')) {
+    var temp =
```

Comment:
Generally with JS we want to use `let` or `const` - there's a bunch of weird scoping issues with `var`